Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.9-bullseye
```

Comment:
```suggestion
FROM python:3.9-bullseye-slim
```

Reply:
Nice idea, but unfortunately (even with the fixed name `3.9-slim-bullseye`) that doesn't work -- the packages fail to install due to needing `gcc`, which isn't present. I'm not sure why that's needed (since the relevant packages do seem to ship wheels and we do have `wheel` installed), but it seems to be the case.

Reply:
https://github.com/srobo/discord-gated-entry/pull/13 enables this.